### DISCUSSION TITLE
Aptos-simple-coin 📒 笔记

### DISCUSSION THREAD

**Discussion by Demian101** [NonceGeek/Move-Camp#204] [Materials]
点击下载： [Aptos-simple-coin.pdf](https://github.com/NonceGeek/Web3-dApp-Camp/files/9960803/Aptos-simple-coin.pdf)

主要内容：
1. Aptos-simple-coin 步骤
2. aptos-framework::coin 源码分析

- [Aptos cli](#aptos-cli)
  - [1. 安装 Aptos](#1-安装-aptos)
  - [2. 启动 local-testnet](#2-启动-local-testnet)
  - [3. 生成测试账户](#3-生成测试账户)
  - [4. 领取 Faucet 激活账户](#4-领取-faucet-激活账户)
  - [5. Test first contract：](#5-test-first-contract)
- [发币 ①：managed_coin 方法](#发币-managed_coin-方法)
  - [1. publish 发布](#1-publish-发布)
  - [2.  initialize](#2--initialize)
  - [3. Register](#3-register)
  - [4. mint](#4-mint)
- [发币 ②：Custom Coin 自定义代币](#发币-custom-coin-自定义代币)
  - [Frontend](#frontend)
  - [test](#test)
    - [issue](#issue)
    - [Transfer](#transfer)
- [1. 项目代码分析](#1-项目代码分析)
  - [1.1 自定义发布源码](#11-自定义发布源码)
  - [1.2 issue 流程](#12-issue-流程)
  - [1.3 register](#13-register)
- [2. aptos::coin 源码分析](#2-aptoscoin-源码分析)

**Comment by leeduckgo** [comment]
这个好棒呀！
